python/scripts/meta.json

```diff
@@ -2083,6 +2083,558 @@
+                "display_text": "",
+                "source_name": "property_type",
+                "sql_name": "property_type",
+                "type": "text"
+            },
+            {
+                "display_name": "property_type_name",
+                "display_text": "",
+                "source_name": "property_type_name",
+                "sql_name": "property_type_name",
+                "type": "text"
+            },
+            {
+                "display_name": "purchaser_type",
+                "display_text": "",
+                "source_name": "purchaser_type",
+                "sql_name": "purchaser_type",
+                "type": "text"
+            },
+            {
+                "display_name": "purchaser_type_name",
+                "display_text": "",
+                "source_name": "purchaser_type_name",
+                "sql_name": "purchaser_type_name",
+                "type": "text"
+            },
+            {
+                "display_name": "respondent_id",
+                "display_text": "",
+                "source_name": "respondent_id",
+                "sql_name": "respondent_id",
+                "type": "text"
+            },
+            {
+                "display_name": "sequence_number",
+                "display_text": "",
+                "source_name": "sequence_number",
+                "sql_name": "sequence_number",
+                "type": "text"
+            },
+            {
+                "display_name": "state_code",
+                "display_text": "",
+                "source_name": "state_code",
+                "sql_name": "state_code",
+                "type": "text"
+            },
+            {
+                "display_name": "state_abbr",
+                "display_text": "",
+                "source_name": "state_abbr",
+                "sql_name": "state_abbr",
+                "type": "text"
+            },
+            {
+                "display_name": "state_name",
+                "display_text": "",
+                "source_name": "state_name",
+                "sql_name": "state_name",
+                "type": "text"
+            },
+            {
+                "display_name": "hud_median_family_income",
+                "display_text": "",
+                "source_name": "hud_median_family_income",
+                "sql_name": "hud_median_family_income",
+                "type": "decimal"
+            },
+            {
+                "display_name": "loan_amount_000s",
+                "display_text": "",
+                "source_name": "loan_amount_000s",
+                "sql_name": "loan_amount_000s",
+                "type": "integer"
+            },
+            {
+                "display_name": "number_of_1_to_4_family_units",
+                "display_text": "",
+                "source_name": "number_of_1_to_4_family_units",
+                "sql_name": "number_of_1_to_4_family_units",
+                "type": "decimal"
```

Comment:
No, I examined a subset of these values as a spreadsheet and some of them are actually decimals.